Update create_graph.py

seigodev opened the pull request:
・node_list2node_dict(): ノードの属性値を辞書形式に変換する。
・create_dependence_graph(): グラフの作成を行う。ノードやエッジのグラフへの作成を行う。
・main()の追加部分: グラフ描画のためのファイルの作成を行う。

seigodev committed:
Add draw graph functions.

・node_list2node_dict(): ノードの属性値を辞書形式に落とし込む。
・create_dependence_graph(): グラフの作成を行う。ノードやエッジのグラフへの作成を行う。
・main()の追加部分: グラフ描画のためのファイルの作成を行う。

aabaa commented on demo/create_graph.py:
```diff
@@ -266,6 +266,47 @@ def cut_edge(source, target):
+        node_dict[node_name]["href"] = str(node.href)
+        node_dict[node_name]["x"] = int(node.x)
+        node_dict[node_name]["y"] = int(node.y)
+        node_dict[node_name]["is_dummy"] = bool(node.is_dummy)
```
些細ですが，これだと毎回 node_dict[node_name] が呼び出されるので，以下のほうがベターです
```python
node_dict[node_name] = {
    'href' : node.href,
    'x' : node.x,
    'y' : node.y,
    'is_dummy' : node.is_dummy
}
```
あと，型はキャストしなくても元からあっているのではと思いましたが違いますか？

aabaa commented on demo/create_graph.py:
```diff
@@ -266,6 +266,47 @@ def cut_edge(source, target):
+    return node_dict
+
+
+def create_dependence_graph(node_list, graph):
```
create_dependency_graph

seigodev committed:
Fix requested changes.

node_list2node_dictのnode_dictへの入力式の書き換え。(284-289行)
create_dependence_graph() -> create_dependency_graph()  (293, 269行)
